### PR TITLE
add Nitay Neeman

### PR DESCRIPTION
**Pull request by nitayneeman**
Hey,

Adding myself 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # IsraelTechWriters
 This repo aim to gather all Israelis tech writer, feel free to open Pull Request and add your self❤️
-Attencion, please add ": " and than with comma separator the topics covered on the blog, and then ". Writing language: " and than with comma separator en / he.
+Attention, please add ": " and than with comma separator the topics covered on the blog, and then ". Writing language: " and than with comma separator en / he.
```
Comment (nitayneeman) [inline README.md:3]: It seems like a type

Comment (nitayneeman) [inline README.md]: It seems like a typo

Comment (MichalPorag) [reply README.md]: @nitayneeman
Please resolve conflicts.

Comment (nitayneeman) [reply README.md]: @MichalPorag done

**Pull request merged**